packages/wallets/ONTO/src/adapter.ts

```diff
@@ -0,0 +1,162 @@
+import {
+    Adapter,
+    BaseSignerWalletAdapter,
+    scopePollingDetectionStrategy,
+    WalletAccountError,
+    WalletName,
+    WalletNotConnectedError,
+    WalletNotReadyError,
+    WalletPublicKeyError,
+    WalletReadyState,
+    WalletSignTransactionError,
+} from '@solana/wallet-adapter-base';
+import { PublicKey, Transaction } from '@solana/web3.js';
+import bs58 from 'bs58';
+
+interface OntoWallet {
+    isOnto?: boolean;
+    signTransaction(transaction: Transaction): Promise<Transaction>;
```

Comment:
Is this used? It seems like the interface below is using `request` with `sol_sign`.

Reply:
@jordansexton Yes,used in Solana Dapp and App Wallet interaction.

Reply:
@jordansexton have any questions about PR ? Please check and commit, Thanks

Reply:
@jordansexton have any questions about PR ? Please check and commit, Thanks

Reply:
@jordansexton have any questions about PR ? Please check and commit, Thanks

Reply:
@jordansexton Please check and commit, Thanks

Reply:
@jordansexton Please check and commit, Thanks

Reply:
@jordansexton Please check and commit, Thanks

Reply:
Hi @PieroDaVinci, the issue is that `signTransaction` is declared here, but on line 139 below:
```
const response = await wallet.request({ method: 'sol_sign', params: [transaction] });
```

Since only `wallet.request` seems to be used, I'm wondering if this method declaration is needed.